backend/app/services/channels/dingtalk/handler.py

```
@@ -548,44 +979,314 @@ class ChatPayload:
+                offset=0,
+            )
+            return None
+        else:
+            # Sync response
+            return (
+                f"✅ 任务已提交到云端执行队列\n\n"
+                f"任务 ID: {result.task.id}\n"
+                "任务完成后将收到通知。"
+            )
         for emitter in self._emitters:
             await emitter.emit_chat_bot_complete(*args, **kwargs)
```

Ruff (0.14.14)

[error] Undefined name `args`

(F821)